api/runnerpool/placer_stats.go

```diff
@@ -0,0 +1,59 @@
+package runnerpool
+
+import (
+	"github.com/sirupsen/logrus"
+	"go.opencensus.io/stats"
+	"go.opencensus.io/stats/view"
+	"go.opencensus.io/tag"
+)
+
+var (
+	fullScanCountMeasure     = stats.Int64("lb_placer_fullscan_count", "LB Placer Full RunnerList Scan Count", "")
```

Comment:
This looks like the placer count stats can use tags? the tags key could be something like 'status'?

Reply:
So, on OSS side, typically the tags are fn_appname, fn_path, but for multi-tenant environments, I think there will be no tags. Leaving this to the caller of RegisterPlacerViews().

Reply:
I don't think that fn_appname should be tag.

see https://prometheus.io/docs/practices/naming/
`CAUTION: Remember that every unique combination of key-value label pairs represents a new time series, which can dramatically increase the amount of data stored. Do not use labels to store dimensions with high cardinality (many different label values), such as user IDs, email addresses, or other unbounded sets of values.`

Reply:
Yes, RegisterPlacerViews() leaves that optional. OSS Fn typically has fn_appname/fn_path, which could lead to high cardinality, but unless we do a sweep of all stats and change them, I think fn_appname/fn_path on OSS side is OK. I think this can be done with separate PR to sweep and change all stats...

Reply:
FYI:
https://github.com/fnproject/fn/blob/master/cmd/fnserver/main.go#L25